core/src/main/java/com/scalar/db/api/Put.java

```diff
@@ -395,7 +448,6 @@ public Put withValue(Column<?> column) {
    * @return a map of {@code Value}s
    * @deprecated As of release 3.6.0. Will be removed in release 5.0.0
    */
-  @Deprecated
```

Comment:
Why did you remove this?

Reply:
It is a mistake, thank you for catching it. 🙇

Reply:
Revised in https://github.com/scalar-labs/scalardb/pull/601/commits/0fbaed49b34872454ed6dd5ba2ea01e26b1be963